tests/cases/unittests/session.ts

```diff
@@ -0,0 +1,459 @@
+/// <reference path="..\..\..\src\harness\harness.ts" />
+
+module ts.server {
+	let lastWrittenToHost: string,

```

Comment:
Darn, that was unintentional. I hate tabs. Can we modify the tslint to pick up on this, too?